stagelib/filecopytools.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import os
+import shutil
+
+
+# List only files in dir
+def list_files(path):
+    """ List all files in the path non-recursively. Do not list dirs."""
+    return [f for f in os.listdir(path)
+            if os.path.isfile(os.path.join(path, f))]
+
+
+def mkdir_if_dne(path, mode=0o755):
+    """Make a directory if it does not exist"""
+    if not os.path.isdir(path):
+        os.mkdir(path, mode)
+
+
+def copy_files(filenames, src_path, dst_path, blacklist):
+    """
+    Copy a list of filenames from src to dst. Will overwrite existing files.
+    If any files are in the blacklist, they will not be copied.
+    If the src directory is in the blacklist, the dest path will not be created, and the files will
+      not be copied or processed.
+    """
+    # Adding a trailing "/" if needed to improve output coherency
+    dst_path = os.path.join(dst_path, '')
+    logging.debug('      {:<80}'.format(os.path.join(src_path, '')))
```

Comment:
The actual 80 aligment makes an horrible alignment on the test files.

Reply:
Yeah. I noticed that.